### ISSUE TITLE
.NET 5 Intellisense data needs to be updated 

### ISSUE THREAD

**Issue by vatsan-madhavan** [dotnet/wpf#3445]
https://github.com/dotnet/wpf/blob/be89bbfaded6ad3f0f4143bf1a8dd075a5789715/global.json#L24

.NET 5 still using intellisense data from .NET Core 3.0 docs. This needs to be updated to use .NET 5 docs based data.

@dotnet/dotnet-winforms - [dotnet/winforms](https://github.com/dotnet/winforms) also has this bug I think. 

[winforms/global.json@L22 in 0416d4](https://github.com/dotnet/winforms/blob/0416d4a5574c2c54e18bd288299fee3a6751526b/global.json#L22)
```json
    "dotnet-api-docs_netcoreapp3.0": "0.0.0.1"
```

**Comment by Tanya-Solyanik** [comment]
@vatsan-madhavan  - what is the value for 5.0?

**Comment by vatsan-madhavan** [comment]
There isn't one yet. Once someone creates and uploads the zip bundles containing the right intellisense data to the Azure Blob Store, they can give it whatever version they want. I recommend the next value in the version space, like 0.0.0.2 ;-) 

See https://github.com/dotnet/winforms/blob/master/docs/intellisense.md and https://github.com/dotnet/wpf/blob/master/Documentation/intellisense.md

**Comment by RussKie** [comment]
@carlossanlop is this something you're working on?

**Comment by vatsan-madhavan** [comment]
The blob items in WPF/Winforms repos were invented and owned by WPF. They are only meaningful in these repos because the targets that read them are custom-written for these repos. The zipped blob themselves are organized to match the expectations of these WPF/WinForms specific targets. 

The content may come from docs repo build, but only a subset (specific to WPF/WinForms) of the intellisense XML's from docs-repo the build are xcopied over to created these zip/blobs. 

In other words, these are not part of a generic Arcade facility, or owned by docs team etc. 

I expect WPF would create the new blobs and upload them to the Azure Storage, and both repos would update their versions. 

Ideally, this system should be deprecated/removed in favor of something more standardized provided by Arcade - when such a system becomes available.

**Comment by merriemcgaw** [comment]
In that case making sure that @fabiant3  and @ryalanms  have this on their radar. We definitely want to get something in for 5.0.

**Comment by AdamYoblick** [comment]
What's the due date for this work please? 😄

**Comment by fabiant3** [comment]
RC1 ships next week, so likely RC2?

**Comment by TheFanatr** [comment]
Is there some news on this?

**Comment by carlossanlop** [comment]
@AdamYoblick can give more information, it's my understanding he was working on updating Intellisense for 5.0 in WPF and WinForms. We already updated IntelliSense in Runtime in RC2.

**Comment by AdamYoblick** [comment]
This has been absolutely done for a long time, the issue was just never updated. https://github.com/dotnet/winforms/pull/4097

Going to close out this issue.
